ui/accounting_settings.ts

```diff
@@ -0,0 +1,140 @@
+import { showError, showInfo } from './utils';
```

Comment:
Is the space after `{` and before `}` a typescript style?

Reply:
It is configurable, you can either have it or not, I just picked some stuff from the webui rules.
If you don't like it I can change the rule and fix it.